internal/event/event.go

```diff
@@ -122,3 +124,12 @@ func Get(ctx context.Context) []Event {
 	}
 	return nil
 }
+
+// EncodeRequestToEvent will write the request out in wire protocol and
+// encode it to base64 and store it in an Event object
+func EncodeRequestToEvent(r *http.Request) string {
```

Comment:
What i don't get about this function is:

[if you json encode a []byte you get... base64 output](https://go.dev/play/p/VCI1_kXUbRk)

Why do we need base64 output as string, instead of using []byte? Anyway...